packages/send-type/index.js

```diff
@@ -12,7 +12,8 @@ module.exports = function (res, code=200, data='', headers={}) {
 	let type = obj[TYPE] || res.getHeader(TYPE);
 
 	if (!!data && typeof data.pipe === 'function') {
-		res.setHeader(TYPE, type || OSTREAM);
+		obj[TYPE] = type || OSTREAM;
+		Object.keys(obj).forEach(header => res.setHeader(header, obj[header]));
```

Comment:
```suggestion
		for (k in obj) {
		  res.setHeader(k, obj[k]);
		}
```

Reply:
Was only using Object.keys() for clearer intent but since we have a plain object, and thus there is no prototype chain at play here, the two are essentially equivalent and for…in is shorter, so sure. :)